### DISCUSSION TITLE
GIF are displayed with artifacts

### DISCUSSION THREAD

**Discussion by albatetra** [moononournation/Arduino_GFX#153] [Q&A]
Hi,
all GIF that I generate with several different programs (EZGIF, Photoshop, etc.) are visualized with artifacts (probably issues related with dithering ). Is there any specific standard/parameter that GIF must be created with to avoid these issues? Thanks
Attached is one gif that is displayed with artifacts.
![lastresort](https://user-images.githubusercontent.com/4553641/166517670-de5673b7-1a9b-401f-81c9-e4ce51a8713a.gif)

**Comment by albatetra** [comment]
In addition, this is the debug messages during the execution:
new_table() malloc: 32776
GIF video start
GIF video end
duration: 116, remain: 4294965651 (3702556595.69%)

**Comment by moononournation** [comment]
what is the actual output? any photo or video?

**Reply by albatetra** [reply]
Hi, thanks for your reply. Here you can see a video of the GIF playback on the display.
https://photos.app.goo.gl/R4RsLabQWeWQVTCt7

**Reply by moononournation** [reply]
Are you using transparent background?

**Comment by albatetra** [comment]
No. The gif is not using transparency. These are the information of the GIF I'm using:

GIF Information
---------------
GIF size: 403,863 bytes (394.4kb)
GIF length: 0.6 second(s)
GIF width/height: 240×240 pixels
number of frames: 30
number of colors: 256
loop count: 5

Frame #1:
---------
x: 0
y: 0
width: 240
height: 240
delay: 20ms
disposal: 1
transparent: false
transparency index: 0 (not used)
transparency color: no (not used)

**Reply by moononournation** [reply]
I am using ezhif.com in the example, you may try is it ok.
